.github/actions/okfn-contributors/entrypoint.sh

```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+
+export PATH="$(yarn global bin):$PATH"
+
+all-contributors check --config $GITHUB_WORKSPACE/.all-contributorsrc \
+  | sed -n '2p' \
+  | sed 's/, /\n/g' \
+  | xargs -I {} all-contributors add --config $GITHUB_WORKSPACE/.all-contributorsrc {} doc
```

Comment:
Pelo que entendi estamos adicionando todas pessoas que já tem commits no projeto mas ainda não estão no `.all-contributorsrc` como `doc`, certo?

Nesse caso prponho que o padrão seja `code,doc`.

cc @lguima que levantou a questão lá na issue.
```suggestion
  | xargs -I {} all-contributors add --config $GITHUB_WORKSPACE/.all-contributorsrc {} code
```